### DISCUSSION TITLE
Copilot shortcut hint popover covers cursor or selection in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by jitterbox** [community/community#18534] [Copilot Conversations]
Using Copilot extension 1.27.0.0 for Visual Studio, the shortcut hint popover seems to be always misaligned (too high), and covers the cursor and/or selection when a Copilot suggestion is made. In this example, the hidden cursor is just after the partially covered word "object " (plus a space):

![image](https://user-images.githubusercontent.com/6708273/173457336-004a205e-e55d-4662-8006-b929e09e7842.png)

This is in Visual Studio 2022 Enterprise (17.2.0) on Windows 10 (21H2), with zoom at 100%.

**Comment by kamtar** [comment]
Same issue on VS Community 17.2.4
